spec/spec_helper.rb

```diff
@@ -24,8 +24,20 @@
   config.infer_spec_type_from_file_location!
   config.use_transactional_fixtures = false
 
+  if Spree.solidus_gem_version < Gem::Version.new('2.11')
+    config.extend Spree::TestingSupport::AuthorizationHelpers::Request, type: :system
+
+    config.after(:each, type: :system) do |example|
+      missing_translations = page.body.scan(/translation missing: #{I18n.locale}\.(.*?)[\s<\"&]/)
+      if missing_translations.any?
+        puts "Found missing translations: #{missing_translations.inspect}\n"
+        puts "In spec: #{example.location}"
+      end
+    end
+  end
```

Comment:
Shouldn't we also do:

```
else
  require 'spree/testing_support/translations'
end
```

? Otherwise in Solidus >= 2.11 (and Solidus master) it won't be loaded, isn't it?

Reply:
Wait, I think we need it, but not here in `Rspec.config`. Maybe we should just load the file conditionally at line `#18`?

`require 'spree/testing_support/translations' unless Spree.solidus_gem_version < Gem::Version.new('2.11')`

Reply:
Good for me, update in the last fixup commit